lom/src/plugins/plugins_common/plugins_helper_test.go

```diff
@@ -3,11 +3,12 @@ package plugins_common
 import (
     "context"
     "fmt"
-    "github.com/stretchr/testify/assert"
```

Comment:
In line 20, Please check return code for InitConfigMgr. This could fail for invalid/missing files.